### PR TITLE
Mar kolya/fix test coverage failure

### PR DESCRIPTION
**Pull request by mar-kolya**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -117,7 +117,7 @@ jobs:
     environment:
       - JAVA8_HOME: /usr/lib/jvm/java-8-openjdk-amd64
       # Tests under Java8 should have complete coverage.
-      - TEST_TASK: test latestDepTest jacocoTestCoverageVerification jacocoTestReport
+      - TEST_TASK: test latestDepTest jacocoTestReport jacocoTestCoverageVerification
```
Comment (tylerbenson) [inline .circleci/config.yml]: if this matters or makes a difference, we should ensure there is a proper task dependency declared.

File: dd-java-agent/testing/testing.gradle [hunk 2]
```diff
@@ -2,9 +2,9 @@ apply from: "${rootDir}/gradle/java.gradle"
 
 minimumBranchCoverage = 0.5
 minimumInstructionCoverage = 0.6
-whitelistedInstructionClasses += whitelistedBranchClasses += [
+excludedClassesConverage += [
   'datadog.trace.agent.test.*Assert',
-  'datadog.trace.agent.test.AgentTestRunner.1',
+  'datadog.trace.agent.test.AgentTestRunner.ErrorCountingListener',
```
Comment (tylerbenson) [inline dd-java-agent/testing/testing.gradle]: Everything else seems stylistic.  Was this the main reason it was failing?

File: .circleci/config.yml (hunk 1 above)
Comment (mar-kolya) [reply .circleci/config.yml]: we do have dependency, this is a stylistic change to make sure that tasks are written in order they are presumably executed.

File: dd-java-agent/testing/testing.gradle (hunk 2 above)
Comment (mar-kolya) [reply dd-java-agent/testing/testing.gradle]: yes

**Comment by mar-kolya** [comment]
Thanks!

**Pull request merged**